### ISSUE TITLE
possible to have simlayer with optional modifiers?

### ISSUE THREAD

**Issue by MagicDuck** [yqrashawn/GokuRakuJoudo#135]
hi there,
I have a simlayer that applies shift:
```
  :simlayers {
    :f-mode {:key :f}
  }
...

    {:des "f-mode (rhs shift)" :rules [:f-mode
      [:6 :!T6] [:7 :!T7] [:8 :!T8] [:9 :!T9] [:0 :!T0]
      [:y :!Sy] [:u :!Su] [:i :!Si] [:o :!So] [:p :!Sp] [:backslash :!Sbackslash]
      [:h :!Sh] [:j :!Sj] [:k :!Sk] [:l :!Sl] [:semicolon :!Ssemicolon] [:quote :!Squote]
      [:n :!Sn] [:m :!Sm] [:comma :!Scomma] [:period :!Speriod] [:slash :!Sslash]
    ]}
```
And I am trying to press cmd-shift-i by:
1. holding cmd
2. tapping f+i simultaneously

This does not seem to work. I have tried the following based on random searches of this repo:
```
  :simlayers {
    :f-mode {:key :f :condi { :optional: {["any"]}}}
  }
```
But it does not seem to have any effect.
Is there a way to have simlayer + modifier?
Thanks in advance for any help! :)

**Comment by yqrashawn** [comment]
Hi this is resolved in #157
you can check changelog below for examples
https://github.com/yqrashawn/GokuRakuJoudo/blob/master/CHANGELOG.org#050---2022-01-08

**Comment by MagicDuck** [comment]
sweet, thank you! 🙇
